File: DataSummitSolution/DataSummitUI/src/app/efModels/userInfo.ts
import { UserInfoType } from "./userInfoType";

export class UserInfo {

	UserInfoId: number;
	Value: string;
	CreatedDate?: Date | string;
	Id: string;
	UserInfoTypeId?: number;
	IdNavigation: number;
	UserInfoType: UserInfoType;

	constructor(userInfoId?: number, value?: string, createdDate?: Date | string,
		id?: string, userInfoTypeId?: number, idNavigation?: number,
		userInfoType?: UserInfoType)
	{
		this.UserInfoId = userInfoId || 0;
		this.Value = value || "";
		this.CreatedDate = createdDate || new Date(Date.now()) || null;
		this.Id = id || "";
		this.UserInfoTypeId = userInfoTypeId || 0 || null;
		this.IdNavigation = idNavigation || 0;
		this.UserInfoType = userInfoType || null;
	}
}

File: DataSummitSolution/DataSummitModels/DB/UserInfo.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace DataSummitModels.DB
{
    public partial class UserInfo
    {
        public long UserInfoId { get; set; }
        public string Value { get; set; }
        public DateTime? CreatedDate { get; set; }
        public string Id { get; set; }
        public byte? UserInfoTypeId { get; set; }

        public virtual AspNetUser IdNavigation { get; set; }
        public virtual UserInfoType UserInfoType { get; set; }
    }
}
